src/components/table/TableWithFilter.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div>
+    <el-button @click="resetDateFilter">reset date filter</el-button>
+    <el-button @click="clearFilter">reset all filters</el-button>
+    <el-table
+      ref="filterTable"
+      :data="tableData"
+      style="width: 100%"
+    >
+      <el-table-column
+        prop="date"
+        label="Date"
+        sortable
+        width="180"
+        column-key="date"
+        :filters="[{text: '2016-05-01', value: '2016-05-01'}, {text: '2016-05-02', value: '2016-05-02'}, {text: '2016-05-03', value: '2016-05-03'}, {text: '2016-05-04', value: '2016-05-04'}]"
```

Comment:
@hvitis is it possible to not hard code the filter? this should ideally be determined by the kind of data added to the table